### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dchai76**
Minor README formatting changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
 
-Currently the Nozzle only run as an app in Cloud Foundry.
+Currently the Nozzle only runs as an app in Cloud Foundry.
 
-1 - Download the latest release of Firehose-to-Logdna.
+1. Download the latest release of Firehose-to-LogDNA.
 ```
-    $git clone https://github.com/logdna/logdna-cloudfoundry.git
-    $cd firehose-to-syslog
+    git clone https://github.com/logdna/logdna-cloudfoundry.git
```
Comment (smusali) [inline README.md:14]: are these 4 trailing spaces needed in front of commands?

File: README.md [hunk 2]
```diff
@@ -1,41 +1,41 @@
...
     cf push firehose-to-logdna --no-start
 ```
 
-4 - Set environment variables with cf cli.
+4. Set environment variables with cf cli.
 ```
     cf set-env firehose-to-logdna API_ENDPOINT https://api.[your cf system domain]
     cf set-env firehose-to-logdna DOPPLER_ENDPOINT wss://doppler.[your cf system domain]:443
-    cf set-env firehose-to-logdna INGESTION_KEY [your ingestion key in LogDNA]
+    cf set-env firehose-to-logdna INGESTION_KEY [your LogDNA ingestion key]
```
Comment (smusali) [inline README.md:32]: I'd suggest `LogDNA Ingestion Key` instead of `your LogDNA ingestion key`

**Review by smusali — CHANGES_REQUESTED** [review]
Just few comments

File: README.md (hunk 2 above)
Comment (dchai76) [reply README.md:32]: Can we leave the "your"? It's not a universal key, it's different for each customer and I think "your" makes that clearer.

File: README.md (hunk 1 above)
Comment (dchai76) [reply README.md:14]: I think you mean leading spaces, and I don't know. Let me follow the same format as https://github.com/cloudfoundry-community/firehose-to-syslog/blob/develop/README.md.

File: README.md (hunk 2 above)
Comment (smusali) [reply README.md:32]: Yup - no harm.

File: README.md (hunk 1 above)
Comment (smusali) [reply README.md:14]: this looks better

**Review by smusali — APPROVED** [review]

**Pull request merged**